README.md

```diff
@@ -1,494 +1,301 @@
+The following table lists all extensions built and released in this repository.
+"build script" instead of "released" denotes extensions that can be built with this repo but aren't hosted here.
+Check out the README files of specific extensions for detailed usage instructions and configuration examples.
 
-The compatibility mechanism of sysext images requires a metadata file in the image under `usr/lib/extension-release.d/extension-release.NAME`.
-It needs to contain a matching OS `ID`, and either a matching `VERSION_ID` or `SYSEXT_LEVEL`. Here you can also set `EXTENSION_RELOAD_MANAGER=1` for a systemd daemon reload.
-Since the rapid release cycle and automatic updates of Flatcar Container Linux make it hard to rely on particular OS libraries by specifying a dependency of the sysext image to the OS version, it is not recommended to match by `VERSION_ID`.
-Instead, Flatcar defined the `SYSEXT_LEVEL` value `1.0` to match for.
-You can also use `ID=_any` and then neither `SYSEXT_LEVEL` nor `VERSION_ID` are needed.
-The sysext image should only include static binaries.
+|    Extension     | Availability | Documentation |
+| ---------------- | ------------ | ------------- |
+| `crio`           |  released    | [crio.md](docs/crio.md) |
+| `docker`         |  released    | [docker.md](docs/docker.md) |
+| `docker_compose` |  released    | [docker_compose.md](docs/docker_compose.md) |
+| `falco`          |  released    | [falco.md](docs/falco.md) |
+| `k3s`            |  released    | [k3s.md](docs/k3s.md) |
+| `keepalived`     | build script | [keepalived.md](docs/keepalived.md) |
+| `kubernetes`     |  released    | [kubernetes.md](docs/kubernetes.md) |
+| `nvidia-runtime` |  released    | [nvidia-runtime.md](docs/nvidia-runtime.md) |
+| `ollama`         |  released    | [ollama.md](docs/ollama.md) |
+| `rke2`           |  released    | [rke2.md](docs/rke2.md) |
+| `tailscale`      |  released    | [tailscale.md](docs/tailscale.md) |
+| `wasmcloud`      |  released    | [wasmcloud.md](docs/wasmcloud.md) |
+| `wasmedge`       |  released    | [wasmedge.md](docs/wasmedge.md) |
+| `wasmtime`       |  released    | [wasmtime.md](docs/wasmtime.md) |
 
-Inside the image, binaries should be placed under `usr/bin/` and systemd units under `usr/lib/systemd/system/`.
-While placing symlinks in the image itself to enable the units in the same way as systemd would normally do (like `sockets.target.wants/my.socket` → `../my.socket`) is still currently supported, this is not a recommended practice.
-The recommended way is to ship drop-ins for the target units that start your unit.
-The drop-in file should use the `Upholds=` property in the `[Unit]` section.
-For example, for starting `docker.socket` we would use a drop-in for `sockets.target` placed in `usr/lib/systemd/system/sockets.target.d/10-docker-socket.conf` with the following contents:
 
-```
-[Unit]
-Upholds=docker.socket
-```
+## What's a sysext and how does it extend Flatcar?
```

Comment:
Putting this above the table seems more natural to me. It answers the "what is the project and what it can do" early

Reply:
That's a good point, will restructure.